src/settings.ts

```diff
@@ -601,6 +601,9 @@ export const POSTERPAGE_MORE_INFO_URL_TITLE = "Full abstract";
 
 export const POSTERHALL_POSTER_IS_LIVE_TEXT = "Presenter is online";
 
+export const DEFAULT_POSTER_THUMBNAIL_IMG =
+  "https://firebasestorage.googleapis.com/v0/b/sparkle-ohbm.appspot.com/o/assets%2Fposter_thumbnail_default.png?alt=media&token=fff064bc-0c62-4a4d-8f5e-0bd25dd06680";
```

Comment:
[note] This is ok since it's targeting `env/ohbm`, but if we wanted to merge this into our main code branch we would obviously need to use something different here.